docs/reference/run.md

```diff
@@ -149,7 +150,7 @@ is receiving its standard input from a pipe, as in:
 The operator can identify a container in three ways:
 
 | Identifier type       | Example value                                                      |
-| --------------------- | ------------------------------------------------------------------ |
+|:----------------------|:-------------------------------------------------------------------|
```

Comment:
Anyway; adding `?w=1` to the URL to hide whitespace changes makes it easier to view; https://github.com/docker/cli/pull/290/files?w=1